spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -58,38 +67,79 @@
         expect(enrollment.current_address_matches_id).to eq(current_address_matches_id)
       end
 
-      it 'creates usps enrollment' do
-        proofer = UspsInPersonProofing::Mock::Proofer.new
-        mock = double
-
-        expect(UspsInPersonProofing::Proofer).to receive(:new).and_return(mock)
-        expect(mock).to receive(:request_enroll) do |applicant|
-          expect(applicant.first_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:first_name])
-          expect(applicant.last_name).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:last_name])
-          expect(applicant.address).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:address1])
-          expect(applicant.city).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:city])
-          expect(applicant.state).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:state])
-          expect(applicant.zip_code).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:zipcode])
-          expect(applicant.email).to eq('no-reply@login.gov')
-          expect(applicant.unique_id).to eq(enrollment.unique_id)
-
-          proofer.request_enroll(applicant)
+      context 'transliteration disabled' do
+        let(:usps_ipp_transliteration_enabled) { false }
+
+        it 'creates usps enrollment without using transliteration' do
+          mock_proofer = double(UspsInPersonProofing::Mock::Proofer.class)
```

Comment:
I think we want either the string, or the literal class, because `Mock::Proofer.class` is `Class` which is probably not what want to stub
```suggestion
          mock_proofer = double('UspsInPersonProofing::Mock::Proofer')
```
```suggestion
          mock_proofer = double(UspsInPersonProofing::Mock::Proofer)
```